spec/roles_spec.rb

```diff
@@ -46,7 +46,7 @@
     it "should remove the given enum to the model_instance.roles by re-assigning all roles" do
       model_instance.should_receive(:roles=).with(subject)
       subject.subtract([:foo, :bar])
-      subject.should have(0).roles
+      expect(subject.size).to be 0
```

Comment:
I believe it would need to be `expect(subject.roles.size).to` and preferably also with `eq(0)`